Documentation/observability/metrics.rst

```diff
@@ -664,7 +670,43 @@ Labels
 - ``method`` is the HTTP method of the request/response.
 - ``protocol`` is the HTTP protocol of the request, (For example: ``HTTP/1.1``, ``HTTP/2``).
 - ``status`` is the HTTP status code of the response.
-- ``reporter`` identifies the origin of the request/response. It is set to ``client`` if it originated from the client, ``server`` if it originated from the server, or ``unknown`` if it's origin isn't known.
+- ``reporter`` identifies the origin of the request/response. It is set to ``client`` if it originated from the client, ``server`` if it originated from the server, or ``unknown`` if its origin is unknown.
+
+Options
+"""""""
+
+This metric supports :ref:`Context Options<hubble_context_options>`.
+
+``httpV2``
+~~~~~~~~~~
+
+``httpV2`` is an updated version of the existing ``http`` metrics.
+These metrics should not be enabled at the same time as ``http``.
+
+The main difference is that ``http_requests_total`` and
+``http_responses_total`` have been consolidated, and use the response flow
+data.
+
+Additionally, the ``http_request_duration_seconds`` metric source/destination
+related labels now are from the perspective of the request. In the ``http``
+metrics, the source/destination were swapped, because the metric uses the
+response flow data, where the source/destination are swapped, but in ``httpV2``
+we correctly account for this.
+
+================================= =================================================== ========== ==============================================
+Name                              Labels                                              Default     Description
+================================= =================================================== ========== ==============================================
+``http_requests_total``           ``method``, ``protocol``, ``status``, ``reporter``  Disabled   Count of HTTP requests
+``http_request_duration_seconds`` ``method``, ``reporter``                            Disabled   Quantiles of HTTP request duration in seconds
```

Comment:
I know it's just copied over, so nothing blocking, but imo "quantiles" suggests it's a summary, not a histogram. I would rather write "Histogram of HTTP request duration in seconds" (and same for metrics help).

Reply:
Ideally, I think so.